pkg/processor/test/callfunction/golang/suite.go

```diff
@@ -73,7 +73,7 @@ func (suite *CallFunctionTestSuite) TestCallFunction() {
 			}
 
 			testRequest := httpsuite.Request{
-				RequestBody:    fmt.Sprintf(`{"callee_name": "%s"}`,
+				RequestBody: fmt.Sprintf(`{"callee_name": "%s"}`,
```

Comment:
fmting?

Reply:
yes, linter is not applied (yet) on test files, I will PR a fix for that anytime soon